episodes/chapter4.md

```diff
@@ -0,0 +1,242 @@
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Understand the types of administrative entities in Vantage6
+- Understand the relation between the administrative entities of Vatange6
+- Understand the permission system of Vantage6
+- Understand the default roles in Vantage6
+- Be able to create a new organization using the Vantage6 UI
+- Be able to create a new user using the Vantage6 UI
+- Be able to create a new collaboration using the Vantage6 UI
+- Be able to reset an api key for a node
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Vantage6 entities and their relations
+
+There are several entities in Vantage6, such as users, organizations, tasks, etc. These entities are created by users that have sufficient permission to do so and are stored in a database that is managed by the central server.
+
+The administrative entities in Vantage6 are:
+
+- **User**: a person who can perform certain actions in Vantage6.
+    - A user can only belong to one **organization**.
+    - A user can have multiple **roles**.
+    - A user can create **tasks** for one or more **organizations** within a **collaboration**.
+- **Organization**: an orgarnization to which users belong
+    - An organization can have zero and more **users**.
+    - An organization can have multiple **nodes**.
+- **Collaboration**: a collection of one or more **organizations**.
+    - Each participating **organization** needs a **node** to compute **tasks** for the **collaboration**.
+    - A collaboration can contain zero or more **studies**.
+    - A **study** is a subset of organizations from the collaboration that are involved in a specific research question. By setting up studies, it can be easier to send tasks to a subset of the organizations in a collaboration and to keep track of the results of these analyses.
+- **Role**: a collection of rules that define the permissions of a user.
+    - A user can have multiple roles.
+    - The permissions of the user are defined by the assigned **rules**.
+- **Node**: a server that can execute **tasks**.
+- **Task**: a task that is executed by one or more **nodes**.
+    - A task should produce an algorithm run for each organization involved in the task. The results are part of such an algorithm run.
+    - A task can be part of a **study**.
+
+The following diagram is simplified relations between these entities. A `1-n` relationship means that the entity on the left side of the relationship can have multiple entities on the right side. For instance, a single organization can have multiple Vantage6 users, but a single user always belongs to one organization. There is one `0-n` relationship between roles and organizations, since a role can be coupled to an organization, but it doesn’t have to be. An `n-n` relationship is a many-to-many relationship: for instance, a collaboration can contain multiple organizations, and an organization can participate in multiple collaborations.
```

Comment:
vantage6 is not with a capital V (unless at the beginning of a sentence of course) :-)

Reply:
and note that this is not the only place you wrote it with capital V

Reply:
It's a pity not using V! But all are corrected.